source/plugins/languages/index.mjs

```diff
@@ -78,10 +78,21 @@ export default async function({login, data, imports, q, rest, account}, {enabled
 
       //Indepth mode
       if (indepth) {
+        //Fetch gpg keys
+        const gpg = []
+        try {
+          const {data:keys} = await rest.users.listGpgKeysForUser({username:login})
+          gpg.push(...keys.map(({key_id:id, raw_key:pub, emails}) => ({id, pub, emails})))
```

Comment:
Since emails are accessible, `commits_authoring` could be autofilled with them

Also, it's probably a good idea to import GitHub's public key since it is used when performing online commits